src/boot_O2/gfxprint.c

```diff
@@ -145,34 +184,34 @@ void GfxPrint_Init(GfxPrint* this) {
     this->baseX = 0;
     this->baseY = 0;
     this->color.rgba = 0;
-    this->flag &= ~GFXPRINT_FLAG1;
-    this->flag &= ~GFXPRINT_USE_RGBA16;
-    this->flag |= GFXPRINT_FLAG4;
-    this->flag |= GFXPRINT_UPDATE_MODE;
+    this->flags &= ~GFXP_FLAG_HIRAGANA;
+    this->flags &= ~GFXP_FLAG_RAINBOW;
+    this->flags |= GFXP_FLAG_SHADOW;
+    this->flags |= GFXP_FLAG_UPDATE;
 }
```

Comment:
This is probably too nitpicky, but could you make the spacing consistent with OoT (or even just make a space between color.rgba and this->flags

![image](https://user-images.githubusercontent.com/47598039/170854640-c6c93393-ea1b-4651-96a6-ba9c67a3fb37.png)